intranet/apps/eighth/views/admin/blocks.py

```diff
@@ -68,6 +68,25 @@ def add_block_view(request):
 
             invalidate_model(EighthBlock)
 
+        if request.POST.get("assign_withdrawn", "off") == "on":
+            try:
+                grp = Group.objects.get(name="Withdrawn from TJ")
```

Comment:
I can't quite remember if this is the case, but does this group have a specific group number?

Reply:
Yes, but I don't know how 8PO handles it. I used the name on both the activity and the group name in case either gets deleted and recreated. We could manually reassign the same ID of course, but I think it is clearer for future devs and 8PO to understand that the name needs to match than the ID?

Reply:
Yeah... makes sense.